.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
Is that intentional ?

Reply:
Not sure TBH. Intellij changed it, doesn't look harmful